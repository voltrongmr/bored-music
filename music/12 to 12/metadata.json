{
  "name": "12 to 12.mp3",
  "artist": "sombr",
  "album": "12 to 12",
  "size": 9800512,
  "created": 1758685161.0384076,
  "modified": 1758602376.7123032,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - Today\u2019s Top Hits\\12 to 12.mp3"
}