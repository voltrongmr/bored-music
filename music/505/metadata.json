{
  "name": "505.mp3",
  "artist": "Arctic Monkeys",
  "album": "Favourite Worst Nightmare (Standard Version)",
  "size": 10236785,
  "created": 1758783984.0,
  "modified": 1758759357.6886008,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - I Love You Like A Love Song\\505.mp3"
}