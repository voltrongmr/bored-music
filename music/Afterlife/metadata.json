{
  "name": "Afterlife.mp3",
  "artist": "d4vd",
  "album": "WITHERED",
  "size": 10070055,
  "created": 1758790190.0,
  "modified": 1758765012.0601413,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - Roommates\\Afterlife.mp3"
}