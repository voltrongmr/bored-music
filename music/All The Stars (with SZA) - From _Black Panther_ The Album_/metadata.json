{
  "name": "All The Stars (with SZA) - From _Black Panther_ The Album_.mp3",
  "artist": "Kendrick Lamar/SZA",
  "album": "Black Panther The Album Music From And Inspired By",
  "size": 9323265,
  "created": 1758783984.0,
  "modified": 1758759352.53079,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - I Love You Like A Love Song\\All The Stars (with SZA) - From _Black Panther_ The Album_.mp3"
}