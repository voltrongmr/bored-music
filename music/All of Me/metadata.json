{
  "name": "All of Me.mp3",
  "artist": "John Legend",
  "album": "Love In The Future (Expanded Edition)",
  "size": 10852292,
  "created": 1758783984.0,
  "modified": 1758759356.4000301,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - I Love You Like A Love Song\\All of Me.mp3"
}