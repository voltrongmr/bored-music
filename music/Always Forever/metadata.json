{
  "name": "Always Forever.mp3",
  "artist": "Cults",
  "album": "Static",
  "size": 9243330,
  "created": 1758783984.0,
  "modified": 1758759360.279845,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - I Love You Like A Love Song\\Always Forever.mp3"
}