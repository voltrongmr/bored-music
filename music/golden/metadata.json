{
  "name": "Golden.mp3",
  "artist": "HUNTR/X/EJAE/AUDREY NUNA/REI AMI/KPop Demon Hunters Cast",
  "album": "KPop Demon Hunters (Soundtrack from the Netflix Film)",
  "size": 7902332,
  "created": 1758685161.0796077,
  "modified": 1758602376.4007814,
  "path": "C:\\Users\\Voltr\\Downloads\\SpotiDownloader.com - Today\u2019s Top Hits\\Golden.mp3"
}